src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
In "copy magnet" is wrong also. If you can, change it. You need to use something like this:
```c++
#if defined(Q_OS_WIN) || defined(Q_OS_OS2)
    QApplication::clipboard()->setText(selectedPeerIPPort.join("\r\n"));
#else
    QApplication::clipboard()->setText(selectedPeerIPPort.join("\n"));
#endif
```